src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -341,19 +369,19 @@ private void addKitRequest(Connection conn, String kitTypeName, KitRequestSettin
                                         object = new KitUploadObject(null, obj.get(PARTICIPANT_ID), shortId,
                                                 null, obj.get(SIGNATURE),
                                                 obj.get(STREET1), obj.get(STREET2), obj.get(CITY),
-                                                obj.get(STATE), obj.get(POSTAL_CODE), obj.get(COUNTRY));
+                                                obj.get(STATE), obj.get(POSTAL_CODE), obj.get(COUNTRY), obj.get(PHONE_NUMBER));
                                     }
                                     else if (containsOrderNumber) {
                                         object = new KitUploadObject(obj.get(ORDER_NUMBER), obj.get(PARTICIPANT_ID), null,
                                                 null, obj.get(NAME),
                                                 obj.get(STREET1), obj.get(STREET2), obj.get(CITY),
-                                                obj.get(STATE), obj.get(POSTAL_CODE), obj.get(COUNTRY));
+                                                obj.get(STATE), obj.get(POSTAL_CODE), obj.get(COUNTRY), obj.get(PHONE_NUMBER));
                                     }
                                     else {
                                         object = new KitUploadObject(null, obj.get(PARTICIPANT_ID), shortId,
                                                 obj.get(FIRST_NAME), obj.get(LAST_NAME),
                                                 obj.get(STREET1), obj.get(STREET2), obj.get(CITY),
-                                                obj.get(STATE), obj.get(POSTAL_CODE), obj.get(COUNTRY));
+                                                obj.get(STATE), obj.get(POSTAL_CODE), obj.get(COUNTRY), obj.get(PHONE_NUMBER));
```

Comment:
what if CMI upload doesn't have PHONE_NUMBER

Reply:
Good catch! I changed it to be null